<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab10GUIClass</class>
 <widget class="QMainWindow" name="Lab10GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab10GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="2" colspan="2">
     <widget class="QTabWidget" name="modeTabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="modeA">
       <attribute name="title">
        <string>Mode A</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0" colspan="3">
         <widget class="QListWidget" name="victimsListWidget"/>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QLineEdit" name="placeLineEdit"/>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="AddPushButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="UpdatePushButton">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QLineEdit" name="nameLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="ageLabel">
          <property name="text">
           <string>Age:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="redoButton">
          <property name="text">
           <string>Redo</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+Y</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="DeletePushButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="placeLabel">
          <property name="text">
           <string>Place:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="mainLabel">
          <property name="text">
           <string>Victims:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QLineEdit" name="ageLineEdit"/>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QLineEdit" name="photographLineEdit"/>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="undoButton">
          <property name="text">
           <string>Undo</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+Z</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="photographLabel">
          <property name="text">
           <string>Photograph:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="modeB">
       <attribute name="title">
        <string>Mode B</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="6" column="2">
         <widget class="QPushButton" name="FilterPushButton">
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QListWidget" name="nextListWidget"/>
        </item>
        <item row="6" column="0">
         <widget class="QLineEdit" name="filterPlaceLineEdit"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="placeFilterLabel">
          <property name="text">
           <string>Place of origin:</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0" colspan="3">
         <widget class="QListWidget" name="mylistWidget"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="nameSaveLabel">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="SavePushButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="filteredLabel">
          <property name="text">
           <string>Filtered List:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="filterAgeLineEdit"/>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="savedLabel">
          <property name="text">
           <string>Mylist (when using in memory repository):</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="3">
         <widget class="QListWidget" name="filteredListWidget"/>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="ageFilterLabel">
          <property name="text">
           <string>Age:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLineEdit" name="saveLineEdit"/>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="NextPushButton">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0" colspan="3">
         <widget class="QPushButton" name="MylistPushButton">
          <property name="text">
           <string>MyList</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="undoSaveButton">
          <property name="text">
           <string>Undo</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+Alt+Z</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QPushButton" name="redoSaveButton">
          <property name="text">
           <string>Redo</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+Alt+Y</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Lab10GUI.qrc"/>
 </resources>
 <connections/>
</ui>
